src/main/haskell/kore/src/Kore/Step/Condition/Evaluator.hs

```diff
@@ -13,6 +13,9 @@ module Kore.Step.Condition.Evaluator
 
 import Data.Reflection
```

Comment:
According to our [style guide](https://github.com/kframework/kore/wiki/Coding-Standards#imports), the imports of `Data.Reflection` and `Control.Monad.Reader` should be grouped together with only one line separating that group from the module header (above) and the next import group (below).